Core/Aspects/Exception/ExceptionLogAspect.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Castle.DynamicProxy;
+using WebNetSample.Core.CrossCuttingConcerns.Logging;
+using WebNetSample.Core.CrossCuttingConcerns.Logging.Log4Net;
+using WebNetSample.Core.Utilities.Interceptors;
+
+namespace WebNetSample.Core.Aspects.Exception;
+
+public class ExceptionLogAspect : MethodInterception
+{
+    private LoggerServiceBase _loggerServiceBase;
+
+    public ExceptionLogAspect(Type loggerService)
+    {
+        if (loggerService.BaseType != typeof(LoggerServiceBase))
+        {
+            throw new System.Exception("Wrong validation type!");
+        }
+
+        _loggerServiceBase = (LoggerServiceBase)Activator.CreateInstance(loggerService);
+    }
+
+    protected override void OnException(IInvocation invocation, System.Exception e)
```

Comment:
Well, it is not a good practice to use full paths to your type since it increase code. Try to always add namespace using in file declaration or if you have some duplication class names but from different namespaces - use namespaces aliases